casanovo/denovo/model.py

```diff
@@ -404,9 +402,32 @@ def on_predict_epoch_end(
         if self.out_filename is None:
             return
         empty_token_score = torch.tensor(0.04)
-        with open(self.out_filename, "w") as f_out:
+        with open(self.out_filename, "a") as f_out:
             writer = csv.writer(f_out, delimiter="\t")
```

Comment:
**suggestion:** Similar to above, I think we should use OS-specific line endings:

```python
writer = csv.writer(f_out, delimiter="\t", lineterminator=os.linesep())
```